ovs: Warn user when cannot connect OVS daemon but desired

cathay4t opened the pull request:
When desire state has OVS interface/bridge, we should warn user on
not able to connect OVS daemon socket. If not the verification error
will confuse people on desired OVS interface/bridge not found.

A warning message will emit in this case, for example:

```
[2024-03-26T10:07:22Z WARN  nmstate::query_apply::net_state] Desired
state contains OVS interfaces, but not able to connection OVS daemon at
socket /run/openvswitch/db.sock
```

Since log cannot checked in auto integration test, manually tested with
OVS daemon stopped.

maiqueb reviewed (approved):
Thank you for this.

liangwen12year commented on rust/src/lib/query_apply/net_state.rs:
```diff
@@ -86,6 +89,14 @@ impl NetworkState {
+        if self.interfaces.has_up_ovs_iface() && !ovsdb_is_running() {
+            log::warn!(
+                "Desired state contains OVS interfaces, but not able \
+                to connection OVS daemon at socket {}",
```
s/connection/connect? other than this, everything looks good to me.

cathay4t committed:
ovs: Warn user when cannot connect OVS daemon but desired

When desire state has OVS interface/bridge, we should warn user on
not able to connect OVS daemon socket. If not the verification error
will confuse people on desired OVS interface/bridge not found.

A warning message will emit in this case, for example:

```
[2024-03-26T10:07:22Z WARN  nmstate::query_apply::net_state] Desired
state contains OVS interfaces, but not able to connection OVS daemon at
socket /run/openvswitch/db.sock
```

Since log cannot checked in auto integration test, manually tested with
OVS daemon stopped.

Signed-off-by: Gris Ge <fge@redhat.com>
